spec/models/todaysmenu_spec.rb

```diff
@@ -15,4 +15,54 @@
     other_menu.valid?
     expect(other_menu).to be_invalid
   end
+
+  # TODO: create_hash_todaysmenusメソッドのテストを書く
+  describe "#create_hash_todaysmenus" do
+    let(:user) { create(:user) }
+    let(:todaysmenu) { create(:todaysmenu, cuisine_id: cuisine.id, serving_count: serving_count, user_id: user.id) }
+    let(:cuisine) { create(:cuisine) }
+    let!(:foodstuff) { create(:foodstuff, cuisine_id: cuisine.id, rawmaterial_id: rawmaterial.id, quantity: quantity) }
+    let(:rawmaterial) { create(:rawmaterial) }
+
+    context "todaysmenuが1つ、foodstuffが1つのとき" do
+      let(:quantity) { "1/3" }
+      let(:serving_count) { 1 }
+
+      it "hashで帰ってくること({rawmaterial.id => fs.quantity)" do
+        user = User.find(todaysmenu.user_id)
+        todaysmenus = user.todaysmenus.includes(:cuisine, cuisine: :foodstuffs).search_in_today
+        expect(todaysmenus.create_hash_todaysmenus(todaysmenus)).to eq({ rawmaterial.id => Rational("1/3") })
```

Comment:
こちらの方がいいですかね。他のテストも同様
```suggestion
        expect(todaysmenus.create_hash_todaysmenus(todaysmenus)).to eq({ rawmaterial.id => Rational(quantity) })
```